src/main/java/com/ss/training/utopia/orchestrator/entity/Airport.java

```diff
@@ -0,0 +1,80 @@
+package com.ss.training.utopia.orchestrator.entity;
+
+import java.io.Serializable;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
+ * @author Trevor Huis in 't Veld
+ */
+@Entity
+@Table(name="tbl_airport")
+public class Airport implements Serializable {
+    
+    /**
+	 *  
+	 */
+	private static final long serialVersionUID = 2302957380915149824L;
+
+	@Id
+    @Column(name="airportId")
+	private Long airportId;
+	
+    @Column(name="name")
+    private String name;
+
+    public Long getAirportId() {
+		return airportId;
+	}
+
+	public void setAirportId(Long airportId) {
```

Comment:
I don't have this setter in my airport class.

Reply:
removed it